core/controllers/base.py

```diff
@@ -277,7 +297,14 @@ def render_json(self, values):
         self.response.write('%s%s' % (feconf.XSSI_PREFIX, json_output))
 
     def render_downloadable_file(self, values, filename, content_type):
-        """Prepares downloadable content to be sent to the client."""
+        """Prepares downloadable content to be sent to the client.
+
+        Args:
+            values: dict. The key-value pairs to encode 
+                in the downloadable file response.
+            filename: str. Name of file to be given.
```

Comment:
`The name of the file to be downloaded` (Similar change below)

Reply:
done